.github/workflows/pandoc.yml

```diff
@@ -1,7 +1,7 @@
 name: Pandoc eBook Generation
 on:
   workflow_run:
-    workflows: ["Ibis eBook Generation"]
+    workflows: ["Pandoc eBook Generation"]
```

Comment:
This commit broke the workflow, as we can see [here](https://github.com/bobbyiliev/introduction-to-bash-scripting/actions/runs/3225497740), it tries to listen to itself to run, this piece of code was design to trigger the Pandoc Generation after the Ibis Generation finished, to avoid file conflicts between the workflows, I've tested it in my fork, and it was working, after changing it back it should be working.

Reply:
```suggestion
    workflows: ["Ibis eBook Generation"]
```

Reply:
Thank you for this clarification! That was totally my bad! Just committed your recommended change!